idea/codeStyleConfig.xml

```diff
@@ -0,0 +1,127 @@
+<code_scheme name="Spring gRPC" version="173">
+	<option name="AUTODETECT_INDENTS" value="false"/>
+	<option name="OTHER_INDENT_OPTIONS">
+		<value>
+			<option name="USE_TAB_CHARACTER" value="true"/>
+		</value>
+	</option>
+	<option name="CLASS_COUNT_TO_USE_IMPORT_ON_DEMAND" value="50"/>
+	<option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="500"/>
+	<option name="IMPORT_LAYOUT_TABLE">
+		<value>
+			<package name="java" withSubpackages="true" static="false"/>
+			<emptyLine/>
+			<package name="javax" withSubpackages="true" static="false"/>
+			<emptyLine/>
+			<package name="" withSubpackages="true" static="false"/>
+			<emptyLine/>
+			<package name="org.springframework" withSubpackages="true" static="false"/>
+			<emptyLine/>
+			<package name="" withSubpackages="true" static="true"/>
+		</value>
+	</option>
+	<option name="RIGHT_MARGIN" value="90"/>
+	<option name="ENABLE_JAVADOC_FORMATTING" value="false"/>
+	<option name="JD_ALIGN_PARAM_COMMENTS" value="false"/>
+	<option name="JD_ALIGN_EXCEPTION_COMMENTS" value="false"/>
+	<option name="JD_KEEP_EMPTY_LINES" value="false"/>
+	<GroovyCodeStyleSettings>
+		<option name="CLASS_COUNT_TO_USE_IMPORT_ON_DEMAND" value="500"/>
+		<option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="500"/>
+		<option name="IMPORT_LAYOUT_TABLE">
+			<value>
+				<emptyLine/>
+				<package name="javax" withSubpackages="true" static="false"/>
+				<package name="java" withSubpackages="true" static="false"/>
+				<emptyLine/>
+				<package name="" withSubpackages="true" static="false"/>
+				<emptyLine/>
+				<package name="org.springframework" withSubpackages="true"
+				         static="false"/>
+				<emptyLine/>
+				<package name="" withSubpackages="true" static="true"/>
+			</value>
+		</option>
+	</GroovyCodeStyleSettings>
+	<JavaCodeStyleSettings>
+		<option name="CLASS_NAMES_IN_JAVADOC" value="3"/>
+		<option name="INSERT_INNER_CLASS_IMPORTS" value="true"/>
+		<option name="CLASS_COUNT_TO_USE_IMPORT_ON_DEMAND" value="50"/>
+		<option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="500"/>
+		<option name="PACKAGES_TO_USE_IMPORT_ON_DEMAND">
+			<value/>
+		</option>
+		<option name="IMPORT_LAYOUT_TABLE">
```

Comment:
These import order are in line w/ the ones in the checkstyle which allows us to easily optmize imports from w/in IDEA and have them match the checkstyles. 

I am sure there is something similar for other editors. I created a wiki page that tells how to do this for IDEA. W/ a TODO section for other editors.